geomap/src/components/LeafletMap.vue

```diff
@@ -56,24 +59,67 @@ import {
 import MarkerCluster from "./MarkerCluster.vue";
 import { Vue } from "vue-class-component";
 import { useStore } from "vuex";
+import L from "leaflet";
+import { Alarm, Node } from "@/types";
 import { Coordinates } from "@/types";
 
 let leafletReady = ref(false);
 let leafletObject = ref("");
 let visible = ref(false);
 let map: any = ref();
 const store = useStore();
+let iconPath;
 
 let center = computed(() => {
-  const coordinates: Coordinates = store.getters['mapModule/getMapCenter']
+  const coordinates: Coordinates = store.getters["mapModule/getMapCenter"];
```

Comment:
Is there a particular reason you're changing the double-quotes to single-quotes in a number of places in this file?

Reply:
This may have happened after formatting the document.